src/cms/forms.py

```diff
@@ -46,6 +55,13 @@ def __init__(self, *args, **kwargs):
 
         self.fields['top_level_nav'].queryset = top_level_nav_items
 
+        if request.journal:
```

Comment:
Consider using self.fields.pop as we do in `core.forms`.
https://github.com/BirkbeckCTP/janeway/blob/d2b5ea7d31fe7d42c43795d208352c4e7c27f60d/src/core/forms.py#L139